en/telemetry/crsf_telemetry.md

```diff
@@ -0,0 +1,199 @@
+
+   ```sh
+   make ark_fmu-v6x_default boardconfig
+   ```
+
+1. In the PX4 board config tool:
+
+   - Disable the default `rc_input` module
+     1. Navigate to the `drivers` submenu, then scroll down to highlight `rc_input`.
+     1. Use the enter key to remove the `*` from `rc_input` checkbox.
+   - Enable the `crsf_rc` module
+     1. Scroll to highlight the `RC` submenu, then press enter to open it.
+     1. Scroll to highlight `crsf_rc` and press enter to enable it.
+
+   Save and exit the PX4 board config tool.
+
+1. [Build the PX4 source code](../dev_setup/building_px4.md) with your changes (again assuming you are using ARKV6X):
+
+   ```sh
+   make ark_fmu-v6x_default
+   ```
+
+This will build your custom firmware, which must now be uploaded to your flight controller.
+
+### Firmware Upload
+
+To upload the custom firmware, first connect your flight controller to the development computer via USB.
+
+You can upload firmware as part of the build process using the `upload` options:
+
+```sh
+make ark_fmu-v6x_default upload
+```
+
+Alternatively you can use QGroundControl to install the firmware, as described in [Firmware > Installing PX4 master, beta, or custom firmware](https://docs.px4.io/main/en/config/firmware.html#installing-px4-master-beta-or-custom-firmware).
+
+### Parameter Configuration
+
+[Set the parameter](../advanced_config/parameters.md) named [RC_CRSF_PRT_CFG](../advanced_config/parameter_reference.md#RC_CRSF_PRT_CFG) to port that is connected to the CRSF receiver (such as `TELEM1`).
+
+This [configures the serial port](../peripherals/serial_configuration.md) to use the CRSF protocol.
+Note that some serial ports may already have a [default serial port mapping](../peripherals/serial_configuration.md#default-serial-port-configuration) or [default MAVLink serial port mapping](../peripherals/mavlink_peripherals.md#default-mavlink-ports) that you will have to un-map before you can assign to CRSF.
+For example, if you want to use `TELEM1` or `TELEM2` you first need to modify [MAV_0_CONFIG](../advanced_config/parameter_reference.md#MAV_0_CONFIG) or [MAV_1_CONFIG](../advanced_config/parameter_reference.md#MAV_1_CONFIG) to stop setting those ports.
+
+There is no need to set the baud rate for the port, as this is configured by the driver.
+
+## TBS Radio Systems
+
+[TBS Crossfire Radio Systems are listed here](https://www.team-blacksheep.com/shop/cat:cat_crossfire#product_listing).
+A few tested options are listed below.
+
+Transmitters:
+
+- [FrSky Taranis X9D Plus (2)](https://www.frsky-rc.com/product/taranis-x9d-plus-2/) is a popular RC controller that has an external module bay.
+  This can be used with modules that are "JR module bay" compatible.
+  It also runs OpenTX software, which supports CRSF.
+- [Radiomaster TX16S]
```

Comment:
@Karthik-Ranga Was there a specific link to your produce? What transmitter module and receiver did you use? (i can then add them into the list)

Reply:
Thanks very much. I added those into the list. 

> The Reptile Dragon 2 Build was quite helpful. Though we may need to make a separate post or video.

I've linked to it below in the ELRS section. It is already its own post.